### PR TITLE
waxp erc20

### PR DESCRIPTION
**Pull request by danielvo11**

### CODE REVIEW

File: contracts/WAXPERC20.sol [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    uint8 public constant DECIMALS = 8;                         // The number of decimals for display
+
+    /**
+     * See {ERC20-constructor}.
+     */
+
+    function initialize(address escrow) public initializer {
+        ERC20UpgradeSafe.__ERC20_init("WAXP Token", "WAXP");
+        _setupDecimals(DECIMALS);
+        uint256 INITIAL_SUPPLY = 386482894311326596;  // supply specified in base units
```
Comment (danielvo11) [inline contracts/WAXPERC20.sol:21]: @porkchop please verify that initialize value is correct. `INITIAL_SUPPLY` I get from [eosio.token stat table](https://wax.bloks.io/account/eosio.token?loadContract=true&tab=Tables&table=stat&account=eosio.token&scope=WAX&limit=100)

**Comment by danielvo11** [comment]
@porkchop As I read in OpenZeppelin documents, OZ CLI is not being actively developed, they recommend us to Upgrades Plugins instead. https://docs.openzeppelin.com/upgrades-plugins/1.x/migrate-from-cli. Should we use Upgrades Plugins instead?

File: contracts/WAXPERC20.sol (hunk 1 above)
Comment (porkchop) [reply contracts/WAXPERC20.sol:21]: That looks good @danielAlvess

**Review by porkchop — APPROVED** [review]
This looks great @danielAlvess

**Comment by porkchop** [comment]
@danielAlvess yes for sure man I think it does make sense to move to the Upgrades Plugin. Let's merge this one and then upgrade in a separate PR.

**Pull request merged**